controls/src/main/java/org/tweetwallfx/controls/dataprovider/TagCloudDataProvider.java

```diff
@@ -54,19 +53,18 @@ public void processTweet(Tweet tweet) {
         updateTree(tweet);
     }
 
-    
     public void setAdditionalTweetWords(List<Word> newWordList) {
         this.additionalTweetWords = newWordList;
     }
-    
+
     public List<Word> getAdditionalTweetWords() {
         return this.additionalTweetWords;
     }
-    
+
     public List<Word> getWords() {
         return tree.entrySet().stream()
-                    .sorted(COMPARATOR.reversed())
-                    .limit(NUM_MAX_WORDS).map(entry -> new Word(entry.getKey(), entry.getValue())).collect(Collectors.toList());        
+                .sorted(COMPARATOR.reversed())
```

Comment:
Use method handle instead

Reply:
i could but it unfortunately it will require me to give the type resolution a hint about the types of the entries